### PR TITLE
Fix E2E after GH-155 fix to swap order of Definition and Gloss

### PR DESCRIPTION
**Pull request by irahopkinson**
A recent PHP change broke the E2E tests.

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/web-languageforge/258)

### CODE REVIEW

**Comment by Nateowami** [comment]
:lgtm:

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io/reviews/sillsdev/web-languageforge/258#-:-LCceLeX4E-e2fT_gcCI:bnfp4nl)*

**Comment by Nateowami** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io/reviews/sillsdev/web-languageforge/258)*

File: test/app/languageforge/lexicon/shared/editor.page.ts [hunk 1]
```diff
@@ -309,7 +309,7 @@ export class EditorPage {
 
     bubbles: {
       first: element.all(by.css('.dc-entry .commentBubble')).get(1),
-      second: element.all(by.css('.dc-entry .dc-sense .commentBubble')).get(0)
+      second: element.all(by.css('.dc-entry .dc-sense .commentBubble')).get(1)
```
Comment (jasonleenaylor) [inline test/app/languageforge/lexicon/shared/editor.page.ts]: I know this was merged already, but if you notice this comment and have time to satisfy my curiosity can you explain why this line was affected?

**Comment by irahopkinson** [comment]
*[test/app/languageforge/lexicon/shared/editor.page.ts, line 312 at r1](https://reviewable.io/reviews/sillsdev/web-languageforge/258#-LCccz57YfelbqdhgFBT-r1-312:-LCfKAbO4aeQe10q648L:b4hpt67) ([raw file](https://github.com/sillsdev/web-languageforge/blob/e35efb3d7c55942e982d9df613f8753dc2bf3585/test/app/languageforge/lexicon/shared/editor.page.ts#L312)):*
<details><summary><i>Previously, jasonleenaylor (Jason Naylor) wrote…</i></summary><blockquote>

I know this was merged already, but if you notice this comment and have time to satisfy my curiosity can you explain why this line was affected?
</blockquote></details>

Because **Gloss** and **Definition** were reordered by the GH-155 PR, **Definition** is now at index 1 instead of 0. This is completely fragile along with the names `bubbles.first` and `bubbles.second` which can't be right and needs to refactored. But I was only fixing this because it was effecting the branch I was working on and didn't take the time to fix the test properly.

---


*Comments from [Reviewable](https://reviewable.io/reviews/sillsdev/web-languageforge/258)*

**Pull request merged**